docs/ss_slide_005.md

```diff
@@ -1,16 +1,16 @@
 # For continuous CA patterns, step size can be too large _or_ too small
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/smaller_scutium.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/smaller_scutium.gif)
 
 ### This single glider in the _Scutium gravidus_ rule set from the Lenia continuous CA framework becomes unstable at too small (and too large) a step size. 
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/superwide_scutium.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/superwide_scutium.gif)
 
 ### This wide glider in the _Scutium gravidus_ rule set from the Lenia continuous CA framework becomes unstable at too smalle (and too large) a step size. 
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_004.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_006)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_004.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_006)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.